graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/runtime/builtins/JSModuleBlock.java

```diff
@@ -0,0 +1,62 @@
+package com.oracle.truffle.js.runtime.builtins;
+
+import com.oracle.truffle.api.object.DynamicObject;
+import com.oracle.truffle.js.builtins.MapPrototypeBuiltins;
+import com.oracle.truffle.js.builtins.ModuleBlockPrototypeBuiltins;
+import com.oracle.truffle.js.runtime.JSContext;
+import com.oracle.truffle.js.runtime.JSRealm;
+import com.oracle.truffle.js.runtime.objects.JSObjectUtil;
+
+public class JSModuleBlock extends JSNonProxy implements JSConstructorFactory.Default.WithSpecies, PrototypeSupplier {
+
+    public static final JSModuleBlock INSTANCE = new JSModuleBlock();
+
+    public static final String CLASS_NAME = "ModuleBlock";
```

Comment:
same constants defined in `ModuleBlockNode` as well

Reply:
Refactored so that the constants only exist in one file.